scripts/apps/contacts/directives/ContactList.ts

```diff
@@ -148,9 +149,9 @@ export function ContactList(
                 return element.scrollTop + element.offsetHeight + 200 >= element.scrollHeight;
             }
 
-            elem.on('keydown', listComponent.handleKey);
+            scrollElem.on('keydown', listComponent.handleKey);
 
-            elem.on('scroll', handleScroll);
+            scrollElem.on('scroll', handleScroll);
```

Comment:
I think you should also call `scrollElem.off();` on `$destroy` event

Reply:
OK

Reply:
Done